LivingCity/benchmarking/runBenchmarks.py

```diff
@@ -0,0 +1,231 @@
+""" This file contains benchmarking scripts.
```

Comment:
In the plots, I'd prefer that each section show up in order of the sequence. In other words, file output is a bar in the middle, but I'd prefer it to be at the bottom because that's the last thing that is done in the entire process. Does this make sense? I'd like them to show up from top to bottom in the sequence that these processes run in the sim.

Reply:
Sounds good! I'll fix it

Reply:
Done! Please check the edited image on the PR

Reply:
Routes conversion to GPU should be after CH output conv.

Reply:
Done!